src/main/java/edu/ucdavis/dss/ipa/api/components/course/CourseViewController.java

```diff
@@ -538,6 +544,27 @@ public CourseView createMultipleCoursesFromDW(@RequestBody List<SectionGroupImpo
 
 						activityService.saveActivity(activity);
 					}
+
+					if (importAssignments) {
+						for (DwInstructor dwInstructor : dwSection.getInstructors()) {
+							DwPerson dwPerson = dwRepository.getPersonByLoginId(dwInstructor.getLoginId());
+
+							if (dwPerson == null) {
+								log.warn("getPersonByLoginId Response from DW returned null, for criterion = " + dwInstructor.getLoginId());
+								continue;
+							}
+
+							String instructorEmail = dwPerson.getEmail();
+
+							// Find or create an instructor
+							Instructor instructor = instructorService.findOrCreate(dwInstructor.getFirstName(), dwInstructor.getLastName(), instructorEmail, dwInstructor.getLoginId(), workgroupId);
+							instructor = instructorService.save(instructor);
```

Comment:
the instructor returned on 560 should be either a newly created instructor, or one fetched from the database - immediately re-saving it shouldn't actually do anything, correct?